### DISCUSSION TITLE
Exporting with command or script

### DISCUSSION THREAD

**Discussion by odensun1** [snipe/snipe-it#13374] [Q&A]
Hello, I apologize if this question has already been addressed. I tried to look around, but didn't see anything phrased exactly this way. I'm trying to see if there's a way to export the asset list with command or somehow by using the Reports Controller? The reason why is that i'm trying to configure an automated export of the asset list that will run maybe once a day or so. I appreciate any assistance and hope all is well, thanks!

**Comment by TShiremanLCOG** [comment]
If you can use Powershell you may be able to use this: https://github.com/snazy2000/SnipeitPS
That allows you to query (and create or edit) assets in various ways.
There may be a way you can write some type of Laravel command to do so but I am unfamiliar with that approach.
